### PR TITLE
Bug 2044201: Support for DATA_SOURCE_NAME and DATA_SOURCE_NAMESPACE

### PR DESCRIPTION
**Pull request by upalatucci**
**Fixes:**
https://bugzilla.redhat.com/show_bug.cgi?id=2044201

**Analysis / Root cause:** 
Missing support for DATA_SOURCE_NAME and DATA_SOURCE_NAMESPACE

**Solution Description:**
Use DATA_SOURCE_NAME and DATA_SOURCE_NAMESPACE when SRC_PVC_NAME and SRC_PVC_NAMESPACE are not defined

### CODE REVIEW

**Comment by yaacov** [comment]
cc:// @dominikholler @tnisan @gouyang

**Comment by yaacov** [comment]
/retitle Bug 2044201: Support for DATA_SOURCE_NAME and DATA_SOURCE_NAMESPACE

File: frontend/packages/kubevirt-plugin/src/components/create-vm/tabs/boot-source.tsx [hunk 1]
```diff
@@ -22,11 +21,10 @@ type BootSourceProps = {
 
 export const BootSource: React.FC<BootSourceProps> = ({ template, state, dispatch }) => {
   const { t } = useTranslation();
-  const name = getTemplateName(template?.variants[0]);
-  const baseImageName = getParameterValue(
-    template?.variants[0],
-    TEMPLATE_BASE_IMAGE_NAME_PARAMETER,
-  );
+
+  const firstVariant = template?.variants[0];
```
Comment (yaacov) [inline frontend/packages/kubevirt-plugin/src/components/create-vm/tabs/boot-source.tsx:25]: nice !

**Comment by upalatucci** [comment]
hei guys can you please review this? @vojtechszocs @glekner @yaacov @metalice @pcbailey @hstastna @avivtur

**Comment by metalice** [comment]
HI , Did you test it on a cluster with new params? this change will affect a lot :)

**Comment by upalatucci** [comment]
No not yet. I created a template and changed the yaml to use the new parameter

**Pull request merged**